homeassistant/components/anova_sous_vide/sensor.py

```diff
@@ -0,0 +1,116 @@
+"""Support for Anova Sous Vide Sensors."""
+from __future__ import annotations
+
+from anova_wifi import AnovaPrecisionCookerSensor
+
+from homeassistant import config_entries
+from homeassistant.components.sensor import (
+    SensorDeviceClass,
+    SensorEntity,
+    SensorEntityDescription,
+    SensorStateClass,
+)
+from homeassistant.const import UnitOfTemperature, UnitOfTime
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers.entity_platform import AddEntitiesCallback
+from homeassistant.helpers.typing import StateType
+from homeassistant.helpers.update_coordinator import CoordinatorEntity
+
+from .const import ANOVA_CLIENT, ANOVA_FIRMWARE_VERSION, DOMAIN
+from .coordinator import AnovaCoordinator
+
+SENSOR_DESCRIPTIONS: dict[str, SensorEntityDescription] = {
```

Comment:
It looks like never need to look these up by key so this could be a tuple of `SensorEntityDescription` instead